config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/templates/service-accounts.yml

```diff
@@ -19,6 +19,26 @@ roleRef:
   name: cluster-admin
   apiGroup: rbac.authorization.k8s.io
 
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cc-worker-service-account
+  namespace: #@ data.values.system_namespace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cc-worker-service-account-superuser
+subjects:
+- kind: ServiceAccount
+  name: cc-worker-service-account
+  namespace: #@ data.values.system_namespace
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
@jamespollard8 @selzoc does this service account really need cluster-admin?

Reply:
I don't think we've done an audit yet of enumerating all permissions that particular service account needs, which of course we should certainly do soon. Fwiw to the best of my knowledge, that deployment needs at least permissions for CRUD operations on kpack Image CRs and I think maybe also route CRs (specifically when handling jobs to delete orgs/everything under an org)

Reply:
The API server deployment's service account is also still a cluster admin

Reply:
Just found a chore in our backlog to address this: https://www.pivotaltracker.com/story/show/171735131

Reply:
@zankich nope! we will eventually prioritize this: https://www.pivotaltracker.com/story/show/171735131

Can you give us a sense of how urgent the need is to reduce the scope of permissions?